src/libcmd/repl.cc

```diff
@@ -791,6 +808,27 @@ void NixRepl::addAttrsToScope(Value & attrs)
     staticEnv->sort();
     staticEnv->deduplicate();
     notice("Added %1% variables.", attrs.attrs()->size());
+
+    lastLoaded = attrs;
+
+    const int max_print = 10;
```

Comment:
Perhaps something like 20-25? There are a some cases in the flake world where it will be more than 10.

Reply:
In that case the pager can be used. Is the team in agreement on this count? Alternatively, this can be changed in the future. I'd like to get this change in soon because it has been in limbo for a long while.

Reply:
yes, I think 20 would be a good user experience